plugins/uartdmx/UartDmxDevice.cpp

```diff
@@ -94,6 +103,9 @@ void UartDmxDevice::SetDefaults() {
   save |= m_preferences->SetDefaultValue(DeviceMalfKey(),
                                          UIntValidator(8, 1000000),
                                          DEFAULT_MALF);
+  save |= m_preferences->SetDefaultValue(DevicePaddingKey(),
+                                         UIntValidator(24, 512),
```

Comment:
Is this data slots, or including start code? I think in theory it can be less than 24 depending on frame rate etc.

Reply:
This is data slots. Got this number somewhere on net.
Wikipedia:  However, it does require that packets be transmitted so that the leading edges of any two sequential BREAKs must be separated by at least 1204 μs, and receivers must be able to handle packets with break-to-break times as short as 1196 μs.[10] The minimum break-to-break transmit time can be achieved by sending packets that contain at least 24 slots